### PR TITLE
Update code.org/ai blue callout banner

### PR DESCRIPTION
**Pull request by dmcavoy**
Updated the blue banner on code.org/ai to push to registration for the first webinar in the AI PL series so that our google ads will help with conversion before we have the landing page fully up. 

Put the translations in and not sure how to show the actual text but you can see it in the PR.

Also removed the teach AI DCDO

Screenshot 2023-06-27 at 7 12 55 AM

### CODE REVIEW

**Comment by dmcavoy** [comment]
@kelbyhawn once you review this and its passing can you merge it today? Thank you!

File: i18n/locales/source/hourofcode/en.yml
Comment (kelbyhawn) [inline i18n/locales/source/hourofcode/en.yml:1]: @dmcavoy strings need to be added to this file: [/pegasus/sites.v3/hourofcode.com/i18n/en.yml](https://github.com/code-dot-org/code-dot-org/blob/50dc4aa9705cd8ede3aebce379413147bdead1ca/pegasus/sites.v3/hourofcode.com/i18n/en.yml)

Comment (kelbyhawn) [reply i18n/locales/source/hourofcode/en.yml:1]: I'll update these strings now, but for the future use that one.

File: lib/dynamic_config/dcdo.rb [hunk 1]
```diff
@@ -47,7 +47,6 @@ def frontend_config
       'webserial-on-chromeos': DCDO.get('webserial-on-chromeos', true),
       'csta-form-extension': DCDO.get('csta-form-extension', false),
       'pl-launch-hero-banner': DCDO.get('pl-launch-hero-banner', false),
-      'teach-ai-launch-2023': DCDO.get('teach-ai-launch-2023', false)
```
Comment (kelbyhawn) [inline lib/dynamic_config/dcdo.rb:50]: Thanks for removing this! 🙌

**Review by kelbyhawn — APPROVED** [review]
Moved the strings to the right `en.yml` file, but otherwise looks great. Thanks for doing this!

**Here's a screenshot w/ the strings showing:**
Screenshot 2023-06-27 at 9 10 33 AM

**Comment by dmcavoy** [comment]
Oooops! Thank you for fixing that!

**Pull request merged**